cuda-toolfile.spec

```diff
@@ -152,6 +152,21 @@ cat << \EOF_TOOLFILE >%{i}/etc/scram.d/cuda-nvrtc.xml
 </tool>
 EOF_TOOLFILE
 
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/cupti.xml
+<tool name="cupti" version="@TOOL_VERSION@">
+  <info url="https://docs.nvidia.com/cupti/Cupti/index.html"/>
+  <lib name="cupti"/>
+</tool>
+EOF_TOOLFILE
+
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/nvperf.xml
+<tool name="nvperf" version="@TOOL_VERSION@">
+  <info url="https://docs.nvidia.com/cupti/Cupti/index.html"/>
+  <lib name="nvperf_host"/>
+  <lib name="nvperf_target"/>
+</tool>
```

Comment:
@fwyzard  do we need these new tools? A tool with out INCLUDE and LIBDIR directory is of no use. If someone adds a dependency in BuildFile then scram does not know what should be `-L/path` for these libs.

Reply:
the include and libdir should be the same as cuda; should I add them explicitly ?

Reply:
did you check if these libs are linked against `libcuda.so`? if then then just add dependency on `cuda` otherwise I would suggest to add a new `cuda_interface` tool file e.g. something like [a] and depend on it 

[a]
```
cat << \EOF_TOOLFILE >%{i}/etc/scram.d/cuda_intrface.xml
<tool name="cuda_interface" version="@TOOL_VERSION@">
  <info url="https://docs.nvidia.com/cuda/index.html"/>
  <client>
    <environment name="CUDA_INTERFACE_BASE" default="@TOOL_ROOT@"/>
    <environment name="LIBDIR"          default="$CUDA_INTERFACE_BASE/lib64/stubs"/>
    <environment name="INCLUDE"         default="$CUDA_INTERFACE_BASE/include"/>
  </client>
  <flags SKIP_TOOL_SYMLINKS="1"/>
</tool>
EOF_TOOLFILE
```

Reply:
No, according to the output of `ldd` the new libraries do not depend on other cuda libraries.

For the moment I'll just add the include and lib64 explicitly, we can revise it once we understand a bit better the dependencies.